### DISCUSSION TITLE
Commerce 20240505

### DISCUSSION THREAD

**Discussion by Sean880128** [J-tour/J-tour.github.io#25] [Polls]
# Commerce 20240505

Hsinchu Shyrell-liau Shyrguang-vill.
新竹十二寮時光村駐唱歌手商演活動，考量到商演需要比較認真，要唱四個小時，所以至少五個人才出演，成團後我將會再Projects建立專案。

## Info
![image](https://lh5.googleusercontent.com/s5cby3oisyW501BxrJPkCCETJoJj8zmIAzkwwdmYDiMJE3UdRr7T_RB0wppqMXU3o-1pqWXEWcTHcExVr37ZM5IQ47j3Nc1WGkz-5xBHTZlyPhzPrHZJEBgxk-QRCdChVw=w2481)

> 活動海報

_Venue_ 新竹縣[十二寮時光村](https://www.google.com/maps/place/%E5%8D%81%E4%BA%8C%E5%AF%AE%E6%99%82%E5%85%89%E6%9D%91-%E5%80%9F%E5%95%8F%E7%AB%99/@24.6711377,120.9931492,18z/data=!4m14!1m7!3m6!1s0x34684ea2382e31ad:0x1b7d4c5d966791ad!2z5Y2B5LqM5a-u5pmC5YWJ5p2RLeWAn-WVj-ermQ!8m2!3d24.6713448!4d120.9943734!16s%2Fg%2F11c1p6fm6b!3m5!1s0x34684ea2382e31ad:0x1b7d4c5d966791ad!8m2!3d24.6713448!4d120.9943734!16s%2Fg%2F11c1p6fm6b?hl=zh-TW&entry=ttu)
_Time_ 2024/5/5 (Sunday) 13:00—17:00
_Revenue_ NT$ 8,000

**Comment by ShengHaoLo** [comment]
5/4 Participate declination due to Taipei [鯨神盃](https://www.facebook.com/TaipeiDragon/posts/pfbid031Gwkq1u6p3SzuT5g1FyQQHs1J17qqmdVAMe5G7NihGsqkvhapGTFbZV9RsyBvXmsl/)
